### ISSUE TITLE
Failed to execute 'appendChild' on 'Node': parameter 1 is not of type 'Node'

### ISSUE THREAD

**Issue by joseluisq** [Selz/plyr#332]
When video is completed appears this error: 
```
plyr-1.8.12.js:1 Uncaught TypeError: Failed to execute 'appendChild' on 'Node': 
parameter 1 is not of type 'Node'.
```

![captura de pantalla 2016-08-05 a las 18 20 16](https://cloud.githubusercontent.com/assets/1700322/17453019/327f5296-5b3b-11e6-8103-439e9870890c.png)

**Comment by Catlite91** [comment]
+1

**Comment by SamPotts** [comment]
Apologies about this. Fix coming soon. I've just been refactoring some other parts too.

**Comment by SamPotts** [comment]
This should be fixed in v1.9.0 - could you give it a go? Bear in mind there were also several breaking changes so check the [changelog](https://github.com/Selz/plyr/blob/master/changelog.md)
